frame.go

```diff
@@ -438,6 +440,10 @@ func newFramer(r io.Reader, w io.Writer, compressor Compressor, version byte) *f
 	f.header = nil
 	f.traceID = nil
 
+	if lwtExt := findCQLProtoExtByName(cqlProtoExts, lwtAddMetadataMarkKey); lwtExt != nil {
```

Comment:
Perhaps a new function that decorates the framer accordingly after initial construction?

Reply:
Yes, will do, thanks.